File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidInvoiceException.java
package io.muun.apollo.domain.errors;

import io.muun.common.exception.PotentialBug;

public class InvalidInvoiceException extends RuntimeException implements PotentialBug {

    public InvalidInvoiceException(String invoice, Throwable cause) {
        super(invoice, cause);
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/newop/InvalidInvoiceException.kt
package io.muun.apollo.domain.errors.newop

import io.muun.apollo.domain.errors.MuunError
import io.muun.common.exception.PotentialBug

class InvalidInvoiceException(invoice: String, cause: Throwable) : MuunError(
    invoice,
    cause
), PotentialBug
